### PR TITLE
Created workflow and about page

### PR DESCRIPTION
**Pull request by ogoigbe12**
created a new branch called created_workflow-and-about_page in this branch newly created I add more images into the img folder and also added more svg file into the assets folder also made updates to the style.css file and also to the profile.html file also updated the README.md file.

### CODE REVIEW

File: porfile.html [hunk 1]
```diff
@@ -56,6 +56,196 @@ <h1 class="title">
         </div>
 
     </section>
+    <section id="projects">
```
Comment (souad988) [inline porfile.html:59]: Great work @ogoigbe12  you implemented almost all this project requirements perfectly 💪🎉, Only one small thing to mention before approving your work , you need to use grid layout to align you project section container as requested in the requirements :
"In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card."

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @ogoigbe12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔No linter errors
✔Implemented almost all projects requirements as requested.

## Required Changes ♻️
_Check the comments under the review._
_ Make sure to add the acknowledgement section to your README file as you are using macroverse template you need to mention that , also it's better to replace your personel image by screenshots of your project this will make your README look professional 😊 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: porfile.html [hunk 2]
```diff
@@ -56,6 +56,196 @@ <h1 class="title">
...
+        </p>
+        <form class="contact-form">
+            <input type="text" class="input" value="Samuel Ogoigbe" name="name" placeholder="Enter Name">
+            <input type="email" class="input" value="ogoigbe13@gmail.com" name="email" placeholder="Enter Email">
+            <textarea name="description" class="textarea" cols="30" rows="7"
+                placeholder="Write your message here"></textarea>
+            <button type="button" class="contact-form-button">Get in touch</button>
+        </form>
+    </section>
+    <section class="works"></section>
```
Comment (MaganaAsiati) [inline porfile.html:248]: - [ ] [ OPTIONAL] Kindly remove the contact form you will work on it after with more documentation :+1:

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @ogoigbe12  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


clap

## Highlights
- descriptive README file  :heavy_check_mark:👨🏻‍💻
- Your commits should start with a capital letter :heavy_check_mark:👨🏻‍💻


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.
- [ ] [OPTIONAL] kindly  Add a descriptive description in your pull request in future which give more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Cheers and Happy coding!👏👏👏

**Pull request merged**